src/components/infiniteScroll/index.js

```diff
@@ -28,23 +30,32 @@ export default class InfiniteScroll extends React.Component<Props> {
     useWindow: true,
     isReverse: false,
     scrollElement: null,
+    showPTRBefore: null,
   };
 
   scrollListener: Function;
   scrollComponent: Object;
   pageLoaded: number;
   _defaultLoader: React.Node;
+  pullToRefresh: any;
 
   constructor(props: Props) {
     super(props);
-
     this._defaultLoader = props.loader;
     this.scrollListener = this.scrollListener.bind(this);
+    this.pullToRefresh = null;
   }
 
   componentDidMount() {
     this.pageLoaded = this.props.pageStart;
     this.attachScrollListener();
+
+    if (this.props.showPTRBefore) {
+      this.pullToRefresh = PullToRefresh.init({
```

Comment:
@mxstbr what is expected behaviour for pullToRefresh ?? currently it is default to `window.location.reload()`. should we use `setState` to update thread instead ??

Reply:
It should call `this.props.refetch` if it exists, which should be passed in from the threadFeed component via Apollo's `this.props.data.refetch`!

Reply:
@mxstbr what is `mobile` directory in codebase ?? its looks similar to `src`